.changeset/heavy-seahorses-poke.md

```diff
@@ -0,0 +1,102 @@
+---
+'astro': minor
+---
+
+The Content Layer API introduced behind a flag in [4.14.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#4140) is now stable and ready for use.
```

Comment:
```suggestion
The Content Layer API introduced behind a flag in [4.14.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#4140) is now stable and ready for use in Astro v5.0.
```
You could choose a little extra protection here, *juuuust* in case. :smile: